README.md

```diff
@@ -100,30 +162,41 @@ This will take approximately a minute to run, with the majority of the time spen
 }
 ```
 
-And if we query the enclave's services again...
+And if we inspect the enclave we will see the Kurtosis module and the three service IDs which the respective local port bindings:
```

Comment:
And since we're reusing the `demo `enclave, we can make this....



"And if we inspect the enclave again....

```
kurtosis enclave inspect demo
```


...we'll see that our enclave now has the Kurtosis module and three Ethereum nodes with local port bindings:"